packages/experiments/src/components/Card/CardItem/CardItem.test.tsx

```diff
@@ -54,26 +54,33 @@ describe('Card Item', () => {
     expect(tree).toMatchSnapshot();
   });
 
-  it('has the correct margin values when the disableChildPadding prop is set to true', () => {
-    const cardStylesFunc = CardStyles as IStylesFunction<ICardProps, ICardTokens, ICardStyles>;
-    const cardItemStylesFunc = CardItemStyles as IStylesFunction<ICardItemProps, ICardItemTokens, ICardItemStyles>;
+  // @todo: Get this test working.
```

Comment:
A lot of commenting in this file 😞

Reply:
Yes, we realized this test has some troubles that we have to fix before merging this PR.

Reply:
If you prefer, you can disable a test with `xit` and similarly `xdescribe`.

Reply:
I'll leave it for now as @khmakoto is going to be taking a closer look at this test.